src/cursor/cur_backup_incr.c

```diff
@@ -70,6 +70,11 @@ __curbackup_incr_blkmod(WT_SESSION_IMPL *session, WT_BTREE *btree, WT_CURSOR_BAC
         cb->nbits = (uint64_t)b.val;
         WT_ERR(__wt_config_subgets(session, &v, "offset", &b));
         cb->offset = (uint64_t)b.val;
+        WT_ERR(__wt_config_subgets(session, &v, "rename", &b));
```

Comment:
Two (minor) questions.

1. Is there enough similarity between the configuration parsing here and in `ckpt_load_blk_mods` to factor it out into a utility function?
2. It's not part of this PR, but those two locations use slightly different code for identifying a match. E.g., `ckpt_load_blk_mods` checks that the string is not `NULL` before doing the `WT_STRING_MATCH` but this function doesn't. I don't have a good sense what we can/can't count on in these two locations so I thought I would just point this out so you can either confirm that it's OK or update it.

Reply:
I agree but I think it would be sufficiently invasive for this ticket. Based on your comment, I have opened WT-6531 specifically to consider this and refactor it. It is non-trivial because the two functions fill in different data structures that have similar/overlapping similar fields. So there is sufficient work that it deserves its own ticket and would clutter up this change.

Reply:
Makes sense to me.